src/react-native/javascript.ts

```diff
@@ -46,8 +55,20 @@ export async function addSentryInit({ dsn }: { dsn: string }) {
     return;
   }
 
+  if (enableSessionReplay) {
+    clack.log.info(
+      `Session Replay will be enabled with default settings (replaysSessionSampleRate: ${sessionReplaySampleRate}, replaysOnErrorSampleRate: ${sessionReplayOnErrorSampleRate}).`,
+    );
+    clack.log.info(
```

Comment:
L: Doesn't print the blue point before the text, which will make it look more like part of the previous info log. 

````suggestion
    clack.log.message(
````